pkg/pillar/cmd/volumemgr/handlework.go

```diff
@@ -248,6 +256,14 @@ func casIngestWorker(ctxPtr interface{}, w worker.Work) worker.WorkResult {
 	if err != nil {
 		result.Error = err
 		result.ErrorTime = time.Now()
+	} else {
+		if ctx.hvTypeKube {
+			cfg := lookupContentTreeConfig(ctx, status.ContentID.String())
+			if cfg != nil && cfg.IsAppImage {
```

Comment:
Does this imply there might be a race where OciImageName might not be set e.g., due to an error?
What about e.g., the case when the ContentTreeConfig has been deleted by zedagent but volumemgr has not yet completed the delete hence there is still a ContentTreeStatus

Reply:
This code is executed during image creation so I did not understand why zedagent deleting the ContentTreeConfig is a valid case in here ?